src/common/header/components/searchBar.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FormEvent, useState } from "react";
+import classNames from "classnames";
+import SearchBarPopup from "./searchBarPopup";
+import SearchIcon from "@mui/icons-material/Search";
+import { ISearchBarProps } from "../interfaces";
+import style from "../styles/searchBar.module.scss";
+
+const SearchBar: React.FC<ISearchBarProps> = ({ theme, menuOpen }: Props) => {
+  const [searchInput, setSearchInput] = useState("");
```

Comment:
Move states and handlers to controller, look up tagCloud component in [this](https://github.com/projectx-curtains/frontend/pull/24) request